db/migrate/20191018160023_change_form_version_code_to_odk_style.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ChangeFormVersionCodeToOdkStyle < ActiveRecord::Migration[5.2]
+  def up
+    FormVersion.all.order(:created_at).each do |form_version|
+      date = form_version.created_at.strftime("%Y%m%d")
+      revision = 0
+
+      # Set version code using ODK date-revision convention
+      loop do
+        raise "Revision can't be more than 2 digits" if revision >= 100
+        form_version.code = date + revision.to_s.rjust(2, "0")
+        revision += 1
+        break unless FormVersion.find_by(code: form_version.code)
+      end
+
+      form_version.save!
+    end
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
It's not a big deal if someone wants to roll back; should this `raise` or simply noop?